math/sin_test.go

```diff
@@ -0,0 +1,44 @@
+package math
+
+import (
+	"math"
+	"testing"
+)
+
+const epsilonSin = 0.001
```

Comment:
Call it `epsilon` and remove the other `epsilonCos` because they are both the same.

Reply:
Done